src/include/ipxe/settings.h

```diff
@@ -78,6 +78,7 @@ struct setting {
 #define SETTING_AUTH_EXTRA	16 /**< Authentication additional settings */
 #define SETTING_CRYPTO		17 /**< Cryptography settings */
 #define SETTING_MISC		18 /**< Miscellaneous settings */
+#define SETTING_LLDP		15 /**< LLDP settings */
```

Comment:
Numbering needs fix

Reply:
I have removed the `src/include/ipxe/errfile.h.orig` and corrected the SETTING_LINE (15->19)